portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.test.ts

```diff
@@ -0,0 +1,637 @@
+      const bankTwoTeamName = 'Bank Two Payment Officer Team';
+      const bankTwo: BankResponse = {
+        ...aBank(),
+        isVisibleInTfmUtilisationReports: true,
+        paymentOfficerTeam: { emails: [bankTwoEmail], teamName: bankTwoTeamName },
+      };
+
+      jest.mocked(api.getAllBanks).mockResolvedValue([bankOne, bankTwo]);
+
+      jest.mocked(api.getUtilisationReports).mockResolvedValue([aNotReceivedUtilisationReportResponse()]);
+
+      const sendEmailCallback = jest.fn().mockRejectedValueOnce(new Error('Failed to send!')).mockResolvedValue(undefined);
+
+      // Act
+      await sendEmailToAllBanksWhereReportNotReceived({
+        emailDescription: '',
+        sendEmailCallback,
+      });
+
+      // Assert
+      expect(sendEmailCallback).toHaveBeenCalledTimes(2);
+      expect(sendEmailCallback).toHaveBeenCalledWith(bankOneEmail, bankOneTeamName, 'October 2023');
+      expect(sendEmailCallback).toHaveBeenCalledWith(bankTwoEmail, bankTwoTeamName, 'October 2023');
+      expect(console.error).toHaveBeenCalledTimes(1);
+      expect(console.error).toHaveBeenCalledWith(expect.stringContaining('Failed to send'), expect.any(Error));
+    });
+
+    it('sends emails to all banks', async () => {
+      // Arrange
+      jest.mocked(externalApi.bankHolidays).getBankHolidayDatesForRegion.mockResolvedValue([]);
+
+      const validBarclaysEmail = 'valid-barclays-email@example.com';
+      const validBarclaysTeamName = 'Barclays Payment Officer Team';
+      const validBarclaysBank: BankResponse = {
+        ...aBank(),
+        isVisibleInTfmUtilisationReports: true,
+        paymentOfficerTeam: { emails: [validBarclaysEmail], teamName: validBarclaysTeamName },
+      };
+
+      const validHsbcEmail = 'valid-hsbc-email@example.com';
+      const otherValidHsbcEmail = 'another-valid-hsbc-email@example.com';
+      const validHsbcTeamName = 'HSBC Payment Officer Team';
+      const validHsbcBank: BankResponse = {
+        ...aBank(),
+        isVisibleInTfmUtilisationReports: true,
+        paymentOfficerTeam: { emails: [validHsbcEmail, otherValidHsbcEmail], teamName: validHsbcTeamName },
+      };
+
+      jest.mocked(api.getAllBanks).mockResolvedValue([validBarclaysBank, validHsbcBank]);
+
+      jest.mocked(api.getUtilisationReports).mockResolvedValue([aNotReceivedUtilisationReportResponse()]);
+
+      const sendEmailCallback = jest.fn();
+
+      // Act
+      await sendEmailToAllBanksWhereReportNotReceived({
+        emailDescription: '',
+        sendEmailCallback,
+      });
+
+      // Assert
+      expect(sendEmailCallback).toHaveBeenCalledTimes(3);
+      expect(sendEmailCallback).toHaveBeenCalledWith(validBarclaysEmail, validBarclaysTeamName, 'October 2023');
+      expect(sendEmailCallback).toHaveBeenCalledWith(validHsbcEmail, validHsbcTeamName, 'October 2023');
+      expect(sendEmailCallback).toHaveBeenCalledWith(otherValidHsbcEmail, validHsbcTeamName, 'October 2023');
```

Comment:
It might be nice to link these values to some array which we can extract from the bank `paymentOfficerTeam`s defined above, but I also think that might be a lot of work for little gain so I don't feel strongly about it